tests/system/docker-compose.yml

```diff
@@ -76,7 +76,7 @@ services:
 
   # faucet
   faucet:
-    image: "bpfk/faucet:latest"
+    image: "hewisonchris/faucet:pr44"
```

Comment:
Oh I see the issue. Yeah that's a bummer. We should have something to handle this.

Reply:
Will update if next ci runs without error.